modal.css

```diff
@@ -0,0 +1,126 @@
+/* Styling for the modal */
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  padding: 0 10px;
+}
+
+.modal-content {
+  background-color: #fefefe;
+  margin: 15% auto;
+  padding: 20px;
+  border: 1px solid #888;
+}
+
+.modal-content h3 {
+  font-family: 'roboto', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  padding: 20px 0;
+}
+
+.modal-content ul {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  padding: 10px 0;
+}
+
+.modal-content ul li {
+  border: 0.5px #979493 solid;
+  display: flex;
+  font-family: 'poppins', sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #3c3a39;
+  padding: 5px 10px;
+}
+
+/* .modalProjectImg {
+  width: 100%;
+  padding: 20px 0;
+} */
+
+.modal-content p {
+  font-family: 'inter', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+  padding: 20px 0;
+}
+
+.close {
+  color: #67798e;
+  float: right;
+  font-size: 35px;
+  font-weight: bold;
+}
+
+.close:hover,
+.close:focus {
+  color: black;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.modalProjectLinks {
+  display: flex;
+  justify-content: space-between;
+  font-size: 14px;
+}
+
+.modalProjectLinks a {
+  display: inline-block;
+  padding: 15px 25px;
+  background-color: #ff6b00; /* Changed to lowercase */
+  color: white;
+  text-decoration: none;
+  border-radius: 5px;
+  font-weight: bold;
+}
+
+/* Modal desktop */
+@media screen and (min-width: 768px) {
+  .desktopModalDesign {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
```

Comment:
- [ ]  The buttons `live version` and `source code` on the desktop design are not aligned properly 🔢 Kindly refer to the screenshot below 👇🏽 and fix the alignment according to the Figma design ❤️ 

![Screenshot from 2023-07-29 11-43-05](https://github.com/DevKaranJ/DevPortfolio/assets/10439283/d13850ca-2078-4389-93e1-f67dba84fbcf)